jobs/src/main/protobuf/metronome.proto

```diff
@@ -13,6 +13,14 @@ message Label {
   optional string value = 2;
 }
 
+/**
+ * A generic (key, value) pair used in various places for parameters.
+ */
+message Parameter {
+  required string key = 1;
+  required string value = 2;
```

Comment:
shouldn't this be optional?

Reply:
marathon uses the mesos defined `Parameter` in proto which is defined as required . 
https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L2626